src/frontend/typer/oppgave.ts

```diff
@@ -149,17 +152,47 @@ export enum PrioritetFilter {
     LAV = 'Lav',
 }
 
-export enum DokumentType {
-    TEST,
+export enum Dokumenttype {
+    SØKNAD_OM_BARNETRYGD = 'SØKNAD_OM_BARNETRYGD',
```

Comment:
Er dette en ID vi bestemmer selv eller må det stemmer overens med eksisterende dokarkiv-metadata? `dokumentTypeId` i metadata for overgangsstønad og kontantstøtte er `OVERGANGSSTØNAD_SØKNAD` og `KONTANTSTØTTE_SØKNAD`, så i så fall kanskje sjekke om det skal være `BARNETRYGD_SØKNAD`?

Reply:
Dette er bare noe som settes som tittel på dokumentet og ikke type, men kan sjekke opp.